JoinColony/colonyCDapp
# Add support for MoveFunds action

## Description

This PR adds support for the MoveFunds action.
<img width="959" alt="image" src="https://user-images.githubusercontent.com/112586815/221321979-aaf90263-a7de-4be4-a0d6-b924a5a4d7fd.png">

## Testing
1. Create a colony using the wizard.
2. Generate at least one each of Mint Tokens and Create Domain actions.
3. Click on the Test Move Funds button.
4. You'll be redirected to Transaction not found page. Refresh and verify the details shown are the same as the Dapp, e.g.: https://xdai.colony.io/colony/jakub/tx/0x38e7a37245fe91417f473b6150860db0f6e25596c12b0affa44796cef7b8936f.

## TODO

- [x] Clean up the saga
- [x] Wire data into the UI

Resolves #277
